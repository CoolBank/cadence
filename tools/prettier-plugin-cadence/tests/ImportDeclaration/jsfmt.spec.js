run_spec(__dirname, ["cadence"]);