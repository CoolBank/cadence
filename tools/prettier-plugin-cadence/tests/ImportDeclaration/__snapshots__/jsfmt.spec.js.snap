// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import.cdc 1`] = `
import Simple    from      0x01
import      Second from   "./cadence/second.cdc"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import Simple from 0x1
import Second from "./cadence/second.cdc"



`;
